packages/kbn-test/src/auth/session_manager.ts

```diff
@@ -54,14 +54,14 @@ export class SamlSessionManager {
   private readonly cloudUsersFilePath: string;
 
   constructor(options: SamlSessionManagerOptions) {
-    this.isCloud = options.isCloud;
```

Comment:
>[!TIP]
> Optional, IIUC, since the `validateCloudSetting` function is an instance method it already has access to both `isCloud` and `kbnHost` and doesn't need to accept them via arguments, so maybe something like this would be a bit simpler?
```ts
  constructor(options: SamlSessionManagerOptions) {
    this.isCloud = options.isCloud;
    ...
    this.kbnHost = Url.format(hostOptionsWithoutAuth);
    ...

    this.validateCloudSetting();
  }

  /**
   * Validates if the 'kbnHost' points to Cloud, even if 'isCloud' was set to false.
   */
  private validateCloudSetting() {
    const cloudSubDomain = '.elastic.cloud';
    if (!this.isCloud && this.kbnHost.includes(cloudSubDomain)) {
      throw new Error(
        `SamlSessionManager: 'isCloud' was set to false, but 'kbnHost' appears to be a Cloud instance: ${kbnHost}
Set env variable 'TEST_CLOUD=1' to run FTR against your Cloud deployment`
      );
    }
  }
```

__question:__ can we use `endsWith` instead of `includes`?

Reply:
Thanks for bringing the idea, totally makes sense. I pushed an update with more sub domains in place.